docs/source/tutorials/package.rst

```diff
@@ -19,3 +19,4 @@ you might want to have a look at the two frameworks to understand basic concepts
    package/tutorial_simclr_clothing.rst
    package/tutorial_simsiam_esa.rst
    package/tutorial_custom_augmentations.rst
+   package/tutorial_simclr_detectron2.rst
```

Comment:
I would suggest renaming the file to `tutorial_pretrain_detectron2.rst`

Reply:
@IgorSusmelj I changed the name and also provided the correct path to the dataset.